app/models/subject.rb

```diff
@@ -203,8 +204,10 @@ def download_reviews
     return unless github_client && pull_request?
     reviews = github_client.get(url + '/reviews', since: comments.order('created_at ASC').last.try(:created_at))
     reviews.each do |review|
-        reviews.concat download_comments_for_review(review) if review[:body].empty?
+      if review[:state] == "COMMENTED"
+        reviews.concat download_comments_for_review(review) 
```

Comment:
I believe that users can leave comments when the review is in any state, not just commented

Reply:
Yeah that’s a mental typo.